list.js

```diff
@@ -9,45 +9,60 @@ const liNote = document.querySelector(".note");
 // Buttons
 const btnMainAddNewTask = document.querySelector(".btn-add");
 const btnAddToList = document.querySelector(".btn-add-form");
-const btnCancel = document.querySelector(".btn-cancel");
-const btnMore = document.querySelector(".btn-more");
-
-// Array
-const listArr = [];
-let id = 0;
 
 // Functions
 const clearInput = function () {
   taskNameInput.value = "";
   taskNoteInput.value = "";
 };
 
-// add TASK to the listArr so later you can remove it
-const addListArr = function (toDo, note, id, done, trash) {
-  listArr.push({
-    name: toDo,
-    note: note,
-    id: id,
-    done: false,
-    trash: false,
+const createListEl = function (toDo, note) {
+  // Creating new elements
```

Comment:
Don't use too many comments, instead focus on using the most descriptive names of variables and functions - so the reader will not need any comments :) 

I understand you might need those comments to establish and deeply understand what is happening, hovewer remember that comments are like trashes - use them only when are necessary to understand what is going on in the code (such as in the complex math algorithm)

Reply:
Yeah - I totally understand - you needed them. And it's not the issue in this case. However, remember when creating code in. the future - it's one of the gold rules of clean code :) Descriptive variables names